lib/manager/flux/readme.md

```diff
@@ -0,0 +1,36 @@
+Checks YAML manifests for [Flux](https://fluxcd.io/) `HelmRelease` resources and extracts dependencies for the `helm` datasource.
+
+The `flux` manager will only extract `HelmRelease` resources linked to `HelmRepository` sources.
+`HelmRelease` resources linked to other kinds of sources like `GitRepository` or `Bucket` will be ignored.
```

Comment:
```suggestion
`HelmRelease` resources linked to other sources like `GitRepository` or `Bucket` will be ignored.
```

Reply:
`GitRepository` (and `Bucket`) is a `kind` of Flux source - it's not a source in itself, which is why I phrased it this way.